pageserver/src/metrics.rs

```diff
@@ -1483,12 +1483,18 @@ pub(crate) static DELETION_QUEUE: Lazy<DeletionQueueMetrics> = Lazy::new(|| {
 });
 
 pub(crate) struct WalIngestMetrics {
+    pub(crate) bytes_received: IntCounter,
     pub(crate) records_received: IntCounter,
     pub(crate) records_committed: IntCounter,
     pub(crate) records_filtered: IntCounter,
 }
 
 pub(crate) static WAL_INGEST: Lazy<WalIngestMetrics> = Lazy::new(|| WalIngestMetrics {
+    bytes_received: register_int_counter!(
+        "pageserver_wal_ingest_bytes_received",
+        "Bytes of WAL ingested from safekeepers",
```

Comment:
```suggestion
        "Bytes of WAL ingested from safekeepers. Not aware of reingestion, e.g., after unclean PS shutdown; will count the re-ingested bytes again.",
```

Reply:
should this disclaimer be added to all related counters?

Reply:
alternatively we could leave it out. I doubt anyone would think this one counter has different behaviour than the rest of the system with regards to reingestion.